centers/models/region.py

```diff
@@ -7,6 +10,20 @@ class RegionFirstLayer(models.Model):
     def __unicode__(self):
         return unicode(self.name)
 
+    def centers(self):
```

Comment:
I got the intention from the test. https://github.com/yebimom/yebimom/pull/31/files#diff-6dcb7bc349474e69ce134287a2dce239R117
But I think you need to change method name, either